<launch>
<node name="extract" pkg="image_view" type="extract_images" respawn="false" required="true" output="screen" cwd="ROS_HOME">
<remap from="image" to="/usb_cam/image_raw"/>
</node>
</launch>
